flash/image/segmentation/data.py

```diff
@@ -71,8 +72,19 @@
     from PIL import Image
 else:
 
-    class Image:
-        Image = None
+    class MetaImage(type):
+        def __init__(cls, name, bases, dct):
+            super().__init__(name, bases, dct)
+
+            cls._Image = None
+
+        @property
+        def Image(cls):
+            warn("Mock object called due to missing PIL library. Install PIL using 'pip install Pillow'.")
+            return cls._Image
```

Comment:
looks the same in all files, so why not have it only once?

Reply:
Agree, this could maybe go in `core/utilities/imports.py` that's where our other import warnings are. Can we also switch it to say to install `lightning-flash[image]` rather than PIL